src/features/APIdata.js

```diff
@@ -2,6 +2,8 @@ export const APIUrl = "https://api.themoviedb.org/3"
 
 export const APIKey = "951446c9e47a99cb6ccfb776c60023c7"
 
+export const APISearchMoviesUrl = `${APIUrl}/search/movie?api_key=${APIKey}&query=`
```

Comment:
Maybe something similar to APImovieDetailsUrl function would be better, because in the future we will be passing not only query, but page as well :) 
``` 
export const APImovieSearchUrl = (query) =>
`${APIUrl}/search/movie?api_key=${APIKey}&query=${query}`